t/cached_conditions.t

```diff
@@ -40,14 +41,14 @@ ok($does_change, 'Available actions change over time');
 
 $does_change = 0;
 CHECK_STATE_CHANGE:
-for (my $i = 0; $i < 5; $i++) {
+for (my $i = 0; $i < 10; $i++) {
     # execute a null action to go to the second state
     $wf->execute_action('FORWARD');
     # and go back again
     $wf->execute_action('BACK');
 
     my $curr_actions = join q{, }, sort ( $wf->get_current_actions() );
-    # diag $curr_actions;
+    #    diag "check_state: $curr_actions";
```

Comment:
Change to commented out code?

Why not include `diag` but put in a conditional, like so:

```perl
if ($TEST_VERBOSE) {
    diag "check_state: $curr_actions";
}
```

Reply:
Much better. Yes: a change to commented-out code, because this change ended up confusing the test suite: it depends on the number of times a condition has been evaluated, which this PR changes for nested conditions...